### DISCUSSION TITLE
lwIP_w5500 - power down or save mode

### DISCUSSION THREAD

**Discussion by infrafast** [esp8266/Arduino#8470] [Q&A]
Hello I am using the interface on esp8266.
The use case is that if I have a cable plugged, the ESP will use the cable to establish network connection, if not it it will use wifi interface. Not both at the same time.
Therefore, my goal is that when I am  using WiFi, then it "shutdown" the W5500, however I am not sure if calling the ::end() function of the driver will take benefit of the power down mode (see page 42 of the w5500 spec: https://cdn.sparkfun.com/datasheets/Dev/Arduino/Shields/W5500_datasheet_v1.0.2_1.pdf)
Before I dig and start implementing any code, is it already implemented to fully shutdown the interface so it doesnt consume any current  other than cutting the power wire ?
Thanks a lot,
BR
Thomas

**Comment by d-a-v** [comment]
>  Before I dig and start implementing any code, is it already implemented to fully shutdown the interface so it doesnt consume any current other than cutting the power wire ?

There is currently no more than [what you already saw](https://github.com/esp8266/Arduino/blob/9f536e68f8f3aa3ffae26493dd0a31e1957d2e81/libraries/lwIP_w5500/src/utility/w5500.cpp#L324-L333)

You are welcome to improve the driver !
